toolkits/global/packages/global-meta/package.json

```diff
@@ -1,9 +1,9 @@
 {
   "name": "@springernature/global-meta",
-  "version": "6.0.2",
+  "version": "6.0.1",
```

Comment:
I think we will need to go with `6.0.3` here as `6.0.2` has been published to npm already so any consumer with `global-meta: "^6.0.0"` in their package.json would get `6.0.2` unless we make this `6.0.3`

Reply:
D'oh!

Reply:
```suggestion
  "version": "6.0.3",
```

Reply:
D'oh, again. I updated the HISTORY.md, not the version.